auth-web/src/components/auth/AccountInfo.vue

```diff
@@ -7,36 +7,36 @@
       <v-alert type="error" class="mb-6" v-show="errorMessage">
         {{ errorMessage }}
       </v-alert>
-
-      <ul class="nv-list">
-        <li class="nv-list-item mb-9">
+      <ul class="nv-list" v-show="!anonAccount">
+        <li class="nv-list-item mb-10">
           <div class="name" id="accountType">Account Type</div>
           <div class="value" aria-labelledby="accountType">
-            <div class="value__title">{{ isPremiumAccount ? 'PREMIUM' : 'BASIC' }}</div>
-            <ul class="bcol-acc__meta mt-1" v-if="isPremiumAccount && currentOrgPaymentSettings">
+            <div class="value__title">{{ isPremiumAccount ? 'Premium' : 'Basic' }}</div>
+            <div>
+              <router-link :to="editAccountUrl">Change account type</router-link>
+            </div>
+          </div>
+        </li>
+        <li class="nv-list-item mb-12" v-if="isPremiumAccount">
+          <div class="name mb-3" id="accountName">Linked BC Online Account Details</div>
+          <v-alert dark color="primary" class="bcol-acc px-7 py-5">
+            <div class="bcol-acc__name">
+              {{ currentOrganization.name }}
+            </div>
```

Comment:
Would have been better to move this as a separate component, as in future this space will grow as an when new payment types comes in picture. May be something to consider in future tickets.